contracts/core/EntryPointSimulations.sol

```diff
@@ -192,7 +192,7 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
 
     //slightly stricter gas limit than the real EntryPoint
     function _getVerificationGasLimit(uint256 verificationGasLimit) internal pure virtual override returns (uint256) {
-        return verificationGasLimit - 300;
+        return verificationGasLimit - 500;
```

Comment:
Magic number. Let's create a constant, seems like we need to change it again and again.

Reply:
I don't think it is needed at all. and with handleOps for validation, it will soon go away anyway.